docs/book/src/clusterctl/provider-contract.md

```diff
@@ -306,15 +306,17 @@ functioning of all the `clusterctl` features both in single tenancy or multi-ten
 
 Provider authors should be aware that `clusterctl move` command implements a discovery mechanism that considers:
 
-* All the objects of Kind defined in one of the CRDs installed by clusterctl using `clusterctl init`. 
+* All the objects of Kind defined in one of the CRDs installed by clusterctl using `clusterctl init`.
 * `Secret` and `ConfigMap` objects.
-* the `OwnerReference` chain of the above objects.
+* The `OwnerReference` chain of the above objects.
+* Any object with the with the "move" label (`clusterctl.cluster.x-k8s.io/move`) attached to it.
+Note that any child objects that are tied the object mentioned should **also** be tagged with the "move" label if they are to be moved as well.
```

Comment:
Should we move this in the does NOT consider list? Or at lest is it possible to have the note as part of the last bullet of the list so it is clear that this is referred to labeled objects only

Reply:
It kind of looks funky in the review page, but it is already part of the last bullet if you view the file.